src/test/scala/io/shiftleft/js2cpg/cpg/passes/AstCreationPassTest.scala

```diff
@@ -139,23 +138,65 @@ class AstCreationPassTest extends AbstractPassTest {
       tmpReturn.checkProperty(PropertyNames.NAME, "_tmp_0")
     }
 
-    "have correct structure for runtime node" in AstFixture(f"`$${x + 1}`") { cpg =>
+    "have correct structure for untagged runtime node" in AstFixture(s"`$${x + 1}`") { cpg =>
       def method = cpg.method.nameExact(":program")
       method.checkNodeCount(1)
 
       def methodBlock = method.expandAst(NodeTypes.BLOCK)
       methodBlock.checkNodeCount(1)
 
-      def call =
-        methodBlock.expandAst(NodeTypes.CALL).expandAst(NodeTypes.CALL).expandAst(NodeTypes.CALL)
+      def call = methodBlock.expandAst(NodeTypes.CALL)
       call.checkNodeCount(1)
       call.checkProperty(PropertyNames.NAME, "__Runtime.TO_STRING")
+      call.checkProperty(PropertyNames.CODE, s"__Runtime.TO_STRING(`$${x + 1}`)")
```

Comment:
While/if we're changing this - how about we rename the magic method to something like `<operator>.templateString`?

Reply:
You are right I guess. Fixed that with the latest commit.

Reply:
We have to adjust policy either way because the method is now varargs. (Or maybe it always was and I never realized it?)

I can't think of - or find - any passes that need to change.

Reply:
Ok, lets put the renaming into a separate PR later on.
For the method itself nothing has changed semantically. It has always been this way.